lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'yaml'
+
+class Order
+
+  def initialize
+    @dishes = []
+    @total = 0
+  end
+
+  def add(dish)
+    update_total(dish)
+    @dishes << dish
+  end
+
+  def review
+    "Your order:\n"+readable_dishes+"Total price: $"+@total.to_s
+  end
+
+  def dishes
+    @dishes.dup
+  end
+
+  def place(send_text = true)
+    if send_text
+      tokens = YAML::load(File.open('./lib/tokens.yml'))
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls. (https://github.com/bbatsov/ruby-style-guide#double-colons)